week-2-practise/level-2/4-slice.js

```diff
@@ -5,7 +5,8 @@
 // - remove the item at the specified index
 
 function remove(arr, index) {
-  return; // complete this statement
+  if (newNames=names.remove())
```

Comment:
This doesn't look like it will work. Hint: the name of the file should give you an idea what standard function to use.